### ISSUE TITLE
[Linux/Docker-linux] Successful connection, no message

### ISSUE THREAD

**Issue by Shadow00Caster** [rnwood/smtp4dev#62]
I am not sure if v3 is technically working or not given it's noted that it is in heavy development; I have the latest Docker image running and I see the sessions listed but it shows 0 messages and there is not response sent back to the client as I would normally expect with a 250.

I have tested the same build on Windows running as a service and it's working as expected, so this appears to just be something with the Docker image.

**Comment by rnwood** [comment]
Hi. Thanks for reporting this issue. I can't reproduce this with the latest image produced yesterday.  If you refresh the page do you see anything in the messages grid and/or the sessions grid?

**Comment by grimly** [comment]
I could reproduce but with a Docker image of mine using the Linux binaries. 

It looks like the messages to give the options are sent without any code.

This is what I get from a telnet session :

```
S: 220 1ce7e78212c4 smtp4dev ready
C: EHLO localhost
S: 250 Nice to meet you.
S: 8BITMIME
S: SIZE
```

From the old windows version I get a "250-" prefix on all server messages.

**Comment by rnwood** [comment]
Thanks for the info. I can confirm that I get that output too. I've updated the CI process to run the automated tests (rnwood/smtpserver project) on Linux and this revealed a few issues with inconsistent line endings (SMTP protocol should use CRLF everywhere). It also incudes a test failure for the above.

I have already fixed the failing tests, but next need to update smtp4dev to depend on the latest version of rnwood/smtpserver which has these and other changes.

**Comment by Sjord** [comment]
I have this issue as well, with Smtp4dev-osx-x64-v3.0.349-master.zip. Thunderbird gives an error in which it says the mailserver responded with "8BITMIME". The sessions tab shows the following session terminated abnormally:
```
220 laptop9 smtp4dev ready
EHLO laptop9.local
250 Nice to meet you.
8BITMIME
SIZE
MAIL FROM:<s.langkemper@redacted.nl>
250 New message started
```

**Comment by rnwood** [comment]
Resolved in 3.0.392 onwards. Rnwood.Smtp4Server now has the tests running on both Windows and Linux and the issues that were found have been fixed.
